SPECS/mariner-rpm-macros/macros

```diff
@@ -42,9 +42,14 @@
 # This section overrides the default behavior of the build macro to export
 # all of the flags handled by set_build_flags. Once the configure macros
 # are consistently applied this override should be removed.
+#
+# USE_CCACHE: when 'mariner_ccache_enabled' is defined, enable building with ccache
+#
 %__spec_build_template	#!%{__spec_build_shell}\
 %{__spec_build_pre}\
 %{set_build_flags}\
+  %{?mariner_ccache_enabled:PATH="/usr/lib/ccache:$PATH" ; export PATH ;}\
+  %{?mariner_ccache_enabled:CCACHE_DIR="${CCACHE_DIR:-/ccache-dir}" ; export CCACHE_DIR ;}\
```

Comment:
Why do we want to allow any pre-set `CCACHE_DIR` value instead of always using `/ccache-dir`? Won't this prevent us from saving and re-using the cache across builds for cases where that env variable was already set to a different path?

Reply:
Yes, this is a typo. CCACHE_DIR should be hard-coded, will fix